### ISSUE TITLE
Change App Language is not effected in Index Page

### ISSUE THREAD

**Issue by hanokkurian** [friendsofagape/autographa-live#104]
I chanced the app language English to Hindi 
But the index tab still in English.

![4](https://user-images.githubusercontent.com/25005190/69475405-9453fb80-0df2-11ea-9b7b-545a543d6fc3.png)

**Comment by kent-rasmussen** [comment]
Hi all, I finished most of a solution to this, but not sure how you want a pull request, given I don't have permission to push my branch to your repo.
FYI, This is not entirely resolved, as I'm not sure how you all want to move forward with translating books:
I see `book-${bookName.toLowerCase()}` apparently used to get book codes, but this wouldn't be sufficient for those with spaces. Do you have another function to create book name translation labels?
I wrote my own line with a regex, which I understand may not be the way you want to go. (but this currently gets closest)
In any case, there is a conflict using this method with Song of Solomon/Songs:
"book-song-of-songs" is used in the translation strings files
"Song of Solomon" is used in src/util/constants.js
So this one book is not translated still. Let me know how you want to create these labels, and I can harmonize.
![Screenshot from 2022-11-24 21-09-35](https://user-images.githubusercontent.com/60263886/203857962-65d3f03d-ea98-4012-b474-b44ec3d0763c.png)

**Comment by joelthe1** [comment]
Hello @kent-rasmussen, thank you for your work and interest in improving Autographa! We have moved all of our development effort to a new version of the application that is being done here: https://github.com/friendsofagape/autographa

As we inch closer to coming to feature parity, we'll soon archive this repository.

Please let us know your use-case and your involvement with Autographa to better support you in the new application.

**Comment by kent-rasmussen** [comment]
Yeah, I should have seen that I was posting to a stale repo; I had thought it was just a stale bug.

I don't plan on using Autographa myself; rather, I have an interest in helping the people who would (my main project is at https://github.com/kent-rasmussen/azt/).

WRT this particular bug pull request, I took part of a Saturday to do a homeschool lesson on Javascript with my daughter. If we we can find similar punctual needs, I'd like to do this again in the future --but hopefully on a repo where it will make a difference for the project.
